plugin.video.aod/changelog.txt

```diff
@@ -0,0 +1,21 @@
+﻿0.1 
+- Initial Source
+0.2 
+- Bug Fix
```

Comment:
For changelogs i personnaly don't really see the issue as this plugin is intended for Germans only. The language files are of course something different.  My advise though is that you put the latest changes on top in the changelog.
Also consider adding the <news> section in future updates as that will replace the changelog.txt
See http://forum.kodi.tv/showthread.php?tid=250936&pid=2361575#pid2361575  for some info.